.github/actions/vmtest/action.yml

```diff
@@ -41,12 +41,10 @@ runs:
     - name: Prepare to build BPF selftests
       shell: bash
       run: |
-        echo "::group::Prepare buidling selftest"
+        echo "::group::Prepare buildling selftest"
         cd .kernel
         cat tools/testing/selftests/bpf/config \
-            tools/testing/selftests/bpf/config.${{ inputs.arch }} \
-            ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest \
-            ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest.${{ inputs.arch }} 2> /dev/null > .config && :
+            tools/testing/selftests/bpf/config.${{ inputs.arch }} > .config
         make olddefconfig && make prepare
         cd -
         echo "::endgroup::"
```

Comment:
not related to this change. can we reuse helpers to start / end group in here?

Reply:
I believe we can, yes. I'll do it as a follow up change.